### DISCUSSION TITLE
chia cpu 100%, SSD 20%

### DISCUSSION THREAD

**Discussion by jefersonrosa10** [Chia-Network/chia-blockchain#5537] [Support]
good afternoon, when i enter china it consumes 100% of the CPU and 60% of the memory only 30% of the SSD, the ssd are too slow to plot and Nmve is spending more than 20 hours for 1 plot, wait for me to help you with that ?

![image](https://user-images.githubusercontent.com/84423114/118847493-29c73300-b8a4-11eb-9d31-07270bef224e.png)

**Comment by BrandtH22** [comment]
@jefersonrosa10  , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
